components/skeleton/test/skeleton-group.visual-diff.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+		<script>
+			const rtl = (window.location.search.indexOf('dir=rtl') !== -1);
+			if (rtl) document.documentElement.setAttribute('dir', 'rtl');
+		</script>
+		<script type="module">
+			import '../../typography/typography.js';
+			import '../../collapsible-panel/collapsible-panel.js';
+			import '../../collapsible-panel/collapsible-panel-group.js';
+			import '../demo/skeleton-group-wrapper.js';
+		</script>
+		<style>
+			.visual-diff {
+				width: 300px;
+			}
+		</style>
+		<title>d2l-skeleton-group</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+	</head>
+	<body class="d2l-typography">
+		<div class="visual-diff" id="all-skeleton">
+			<d2l-skeleton-group-test-wrapper>
```

Comment:
Do you know what the plan is in terms of initially adopting skeleton groups in our existing components? Like do we plan to bake it into collapsible-panel, collapsible-panel-group, list, etc? Or yes, but in a separate PR?

Reply:
As discussed in standup, I don't think there are any _immediate_ plans to implement this into core components. And I think any developments on that front would need design considerations.